lib/_stream_writable.js

```diff
@@ -144,6 +145,9 @@ function Writable(options) {
 
     if (typeof options.writev === 'function')
       this._writev = options.writev;
+
+    if (typeof options.end === 'function')
+      this._end = options.end;
```

Comment:
We could but 

- it would be inconsistent  with how the other methods work
- after making the change the next thing I'd have to do is make a regex to undo it for readable-stream